bad_code.java

```diff
@@ -0,0 +1,54 @@
+public class BadCodeExample {
+    
+    public static void main(String[] args) {
+        ChildClass child = new ChildClass();
+        child.doSomething();
+        child.doSomethingElse();
+    }
+}
+
+class ParentClass {
+    public int someNumber = 10;
```

Comment:
Consider adding a constructor to the `ParentClass` to initialize the `someNumber` and `someString` fields, instead of initializing them directly. [medium]